apps/dashboard/app/views/batch_connect/sessions/card/_host.html.erb

```diff
@@ -1,11 +1,13 @@
 <% if session.ssh_to_compute_node? %>
   <p>
     <strong><%= t('dashboard.batch_connect_sessions_stats_host') %></strong>  
-    <%= session.connect.host ? link_to(
-        session.connect.host, OodAppkit.shell.url(
-        host: session.connect.host).to_s, target: "_blank", 
-        class: "btn btn-primary btn-sm fas fa-terminal"
-       ) : t('dashboard.batch_connect_sessions_stats_undetermined_host') %>
+    <% if session.connect.host %>
+      <a href = "<%= OodAppkit.shell.url(host: session.connect.host).to_s %>" target = "_blank" class = "btn btn-primary btn-sm fas fa-terminal"> 
```

Comment:
Do not put spaces inbetween HTML attributes and their values.

This should be `target="_blank"` not `target = "_blank"`

Reply:
IDK if it's actually a part of the HTML spec or what (browsers can likely handle these spaces), it's just as a human you never see spaces here.

Reply:
ah ok, good to know